app/src/main/java/org/fossasia/openevent/app/core/faq/create/CreateFaqViewModel.java

```diff
@@ -0,0 +1,68 @@
+package org.fossasia.openevent.app.core.faq.create;
+
+import android.arch.lifecycle.LiveData;
+import android.arch.lifecycle.MutableLiveData;
+import android.arch.lifecycle.ViewModel;
+
+import org.fossasia.openevent.app.common.ContextManager;
+import org.fossasia.openevent.app.data.event.Event;
+import org.fossasia.openevent.app.data.faq.Faq;
+import org.fossasia.openevent.app.data.faq.FaqRepository;
+import org.fossasia.openevent.app.utils.ErrorUtils;
+
+import javax.inject.Inject;
+
+import io.reactivex.disposables.CompositeDisposable;
+
+public class CreateFaqViewModel extends ViewModel {
+
+    private final FaqRepository faqRepository;
+    private final Faq faq = new Faq();
+    private CompositeDisposable compositeDisposable = new CompositeDisposable();
+    private final MutableLiveData<Boolean> progress = new MutableLiveData<>();
+    private final MutableLiveData<String> error = new MutableLiveData<>();
+    private final MutableLiveData<String> success = new MutableLiveData<>();
+    private final MutableLiveData<Boolean> dismiss = new MutableLiveData<>();
```

Comment:
Use `SingleEventLiveData` which is specifically made for sending one time action such as showing snackbar or redirecting the user or closing something.